### PR TITLE
fix: wallet header spacing to match other tabs

### PR DESCRIPTION
**Pull request by briansztamfater**
fixes ?

From design review during offsite note:
`Wallet header jumps (is higher) when switching tabs`

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Log in
- Go to wallet tab
- Verify top bar spacing matches the rest of the tabs (communities / messages)

status: ready

### CODE REVIEW

File: src/status_im/contexts/wallet/home/style.cljs [hunk 1]
```diff
@@ -23,5 +23,5 @@
 
 (defn home-container
   []
-  {:margin-top (safe-area/get-top)
+  {:margin-top (+ (safe-area/get-top) 8)
```
Comment (briansztamfater) [inline src/status_im/contexts/wallet/home/style.cljs:26]: Matches spacing for the rest of the tabs

https://github.com/status-im/status-mobile/blob/f0026b4d109a97d9b5aec97f4f5db9ecaad3c4e2/src/status_im/common/home/banner/style.cljs#L49

**Comment by J-Son89** [comment]
¡Que rápido! 🏎️

**Review by J-Son89 — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]
🚀

**Comment by mariia-skrypnyk** [comment]
Hi @briansztamfater !

This fix is really quick! 🤌
It would be much easier for QA's in future if you'll add video or tiny details of reproduction 🙏 but I caught this issue! 
Thanks!
@Francesca-G please review it. Issue took place when testnet mode was off.

**Issue**

https://github.com/status-im/status-mobile/assets/147824399/ed910f0c-bb2a-4623-9a8b-b3705a919921

**Review by Francesca-G — APPROVED** [review]
Looks good to me 👍

**Pull request merged**